_posts/2017-07-29-getting-started-with-emacs.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: Getting Started With Spacemacs
+author: aria
+---
+
+[Spacemacs](http://spacemacs.org) is a pre-made configuration for GNU editor Emacs. It combines the superior editing of vim and the extensibility of emacs into one thing,as well as premade configs for many languages.
```

Comment:
nitpick ; could you add links to both vim and emacs official sites? Generally, I think mentioning any kind of other software, service, etc. should be accompanied with an embedded link. If ever you think that someone reading this might be, "wait, wtf is XYZ? i've never heard of it", just include a link.

(I recognize both vim and emacs are meme-status editors)